src/pages/developing/frameworks/vanilla.mdx

```diff
@@ -22,6 +22,9 @@ HTML and SCSS partials to build websites and user interfaces. Adopting the
 library enables developers to use consistent markup, styles, and behavior in
 prototype and production work.
 
+For v10 documents please visit our
+[v10 documentation](https://v10.carbondesignsystem.com/developing/frameworks/vanilla).
+
```

Comment:
Could this instead be in an `<InlineNotification>`? The other framework pages are being updated in #2866 to include a blurb about v11 support in an InlineNotification

Reply:
hah, I didn't see Josh's feedback above. We can use #2866 as a template for the InlineNotification though